lib/sdf-server/src/service/v2/workspace/install_workspace.rs

```diff
@@ -42,9 +42,7 @@ pub async fn install_workspace(
             .tenancy()
             .workspace_pk_opt()
             .ok_or(WorkspaceAPIError::RootTenancyInstallAttempt)?;
-        Workspace::get_by_pk(&ctx, &workspace_pk)
-            .await?
-            .ok_or(WorkspaceAPIError::WorkspaceNotFound(workspace_pk))?
```

Comment:
These WorkspaceAPI errors are interpreted differently by the into_response, so dropping them will break existing behavior

Reply:
I modified the into_response() to look at WorkspaceError::WorkspaceNotFound instead. I also removed the error variant so we can be sure that if anything else interprets that error, it would not compile (everything is clear).

Reply:
I'm definitely willing to keep using WorkspaceAPIError here; I can see us maybe wanting a rule "if we're interpreting the error differently we should have our own local variant." We don't really follow that in other places right now, but that's not a reason not to have the rule :)

Doesn't feel like a decision with a clear answer to me, but I'm on board if you have stronger opinions.